### DISCUSSION TITLE
How can I improve this cifar10Model?

### DISCUSSION THREAD

**Discussion by ajaygithub2** [mrdbourke/pytorch-deep-learning#247] [General]
Hello everyone.

First of all, Daniel, you are the master. If you are reading this, I am so so grateful to you for making this course and making it available on youtube. Everything I have learned about deep learning, I have learned from you. So I really really want to thank you for that. You deserve everything mate.

After completing the video till 19:52 hrs, I tried to work on a dataset called `cifar10`. I have made 3 models for it. I **tried many different things** with the last model, changing many hyperparameters (not randomly) but only with a logic. I was able to get the model train accuracy up to 66% and test accuracy up to 52%. However, **I am not able to improve upon the model**. Any suggestions would be a great help.  There is a big markdown cell somewhere in the end of this notebook which has all the logs for what changes I tried and the results I got. It is a bit messy though.

Here is the link for notebook:
https://colab.research.google.com/drive/1BJphJpC6sabdunb7brUlnAYbbTxvcgu3?usp=sharing

This notebook has comment access and you can also leave a comment in this discussion.

Thankyou for reading this.
